awswrangler/s3/_delete.py

```diff
@@ -146,29 +118,11 @@ def delete_objects(
         last_modified_end=last_modified_end,
         s3_additional_kwargs=s3_additional_kwargs,
     )
-    if len(paths) < 1:
-        return
+
     buckets: Dict[str, List[str]] = _split_paths_by_bucket(paths=paths)
     for bucket, keys in buckets.items():
         chunks: List[List[str]] = _utils.chunkify(lst=keys, max_length=1_000)
-        if len(chunks) == 1:
-            _delete_objects(
-                bucket=bucket, keys=chunks[0], boto3_session=boto3_session, s3_additional_kwargs=s3_additional_kwargs
-            )
-        elif use_threads is False:
-            for chunk in chunks:
-                _delete_objects(
-                    bucket=bucket, keys=chunk, boto3_session=boto3_session, s3_additional_kwargs=s3_additional_kwargs
-                )
-        else:
-            cpus: int = _utils.ensure_cpu_count(use_threads=use_threads)
-            with concurrent.futures.ThreadPoolExecutor(max_workers=cpus) as executor:
-                list(
-                    executor.map(
-                        _delete_objects_concurrent,
-                        itertools.repeat(bucket),
-                        chunks,
-                        itertools.repeat(s3_additional_kwargs),
-                        itertools.repeat(_utils.boto3_to_primitives(boto3_session=boto3_session)),
-                    )
-                )
+        executor = _get_executor(use_threads=use_threads)
```

Comment:
So a `ray_get` was not required in the end? Not sure how this operation would be blocking without it?